data/sea/42-text-conversion.h

```diff
@@ -245,16 +243,16 @@ static ierror_msg_t INLINE fixed_read_itime (const char *p, const size_t size, i
         text_is_digit (p[8]) &&
         text_is_digit (p[9])) {
 
-        const iint_t year  = p[0] * 1000
-                           + p[1] * 100
-                           + p[2] * 10
-                           + p[3];
+        const iint_t year  = (p[0] - '0') * 1000
+                           + (p[1] - '0') * 100
+                           + (p[2] - '0') * 10
+                           + (p[3] - '0');
 
-        const iint_t month = p[5] * 10
-                           + p[6];
+        const iint_t month = (p[5] - '0') * 10
+                           + (p[6] - '0');
 
-        const iint_t day   = p[8] * 10
-                           + p[9];
+        const iint_t day   = (p[8] - '0') * 10
+                           + (p[9] - '0');
```

Comment:
I don't think so, it would have just had very weird dates, this would have been `49 * 10` instead of `1 * 10` for example